### PR TITLE
Build inside container

### PR DESCRIPTION
**Pull request by SK4P3**
Build webpack inside of container to avoid node version issues.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,26 @@
-FROM nginx:alpine
+FROM amd64/nginx:alpine
 MAINTAINER hhendrik@ethz.ch
+MAINTAINER aschnabl@ethz.ch
+
+RUN mkdir /rbb_web
```
Comment (abrandemuehl) [inline Dockerfile]: You don't need this command I think, the COPY will handle it

File: Dockerfile [hunk 2]
```diff
@@ -1,8 +1,26 @@
...
+RUN apk update
+RUN apk add  --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.7/main/ nodejs=8.9.3-r1
+
+RUN node --version
+RUN npm --version
+RUN npm install yarn 
+RUN ./node_modules/yarn/bin/yarn --version
+
+RUN ./node_modules/yarn/bin/yarn install
+RUN ./node_modules/yarn/bin/yarn build-staging
```
Comment (abrandemuehl) [inline Dockerfile:14]: Could you make this into a multi-stage build? It's a feature of docker where you can install deps, build a result then copy the result to a new container that won't copy all the layers from building. That way the `node_modules` won't get brought along for the entire container, also you could base the build container off a node container and the deployment one off nginx

https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds

**Review by abrandemuehl — CHANGES_REQUESTED** [review]
Looks like a good start. I've left some minor comments to improve the build process with docker

File: Dockerfile (hunk 2 above)
Comment (SK4P3) [reply Dockerfile:14]: Done

**Review by abrandemuehl — APPROVED** [review]

**Pull request merged**